src/components/MarkdownProvider/components/PropSheets.tsx

```diff
@@ -6,36 +6,49 @@
  */
 
 import React from 'react';
-import { css } from 'styled-components';
+import styled from 'styled-components';
 import { ComponentDoc } from 'react-docgen-typescript';
-import { getColor } from '@zendeskgarden/react-theming';
-import { MD, Paragraph } from '@zendeskgarden/react-typography';
-import { Table, Head, Body, HeaderRow, HeaderCell, Row, Cell } from '@zendeskgarden/react-tables';
+import { Paragraph, Code } from '@zendeskgarden/react-typography';
+import {
+  Table,
+  Head,
+  Body,
+  HeaderRow,
+  HeaderCell,
+  Row as GRow,
+  Cell
+} from '@zendeskgarden/react-tables';
+import { StyledH4 as Title } from './Typography';
 
-import { IPackage } from './PackageDescription';
-import { StyledH3 } from './Typography';
+const API = styled.div`
+  margin: ${p => p.theme.space.base * 6}px 0;
+`;
 
-export const PropSheets: React.FC<{ data: ComponentDoc[]; reactPackage: IPackage }> = ({
-  data,
-  reactPackage
-}) => {
+const Row = styled(GRow)`
+  font-family: ${p => p.theme.fonts.mono};
+`;
+
+const HeaderContainer = styled.div`
+  margin: 0 0 ${p => p.theme.space.base * 6}px;
+`;
```

Comment:
All styled elements should have a `Styled*` prefix to help determine if they can be referenced by other styled elements.

i.e. `StyledAPI`, `StyledRow`, `StyledHeaderContainer`

If these areas are not reused, it is also more clear to use the inline `css` prop to apply local styles.

`HeaderContainer` could become

```
<div css={css`margin: 0 0 ${p => p.theme.space.lg}px`}></div>
```